### PR TITLE
add rosdep for cpuburn and lm-sensors

### PR DESCRIPTION
**Pull request by mikeferguson**
Adds two rosdeps for commonly used tools for heat/stress testing robot computers.

This might look like a small bit of duplication as there is a key for libsensors4-dev, which happens to use lm_sensors on arch/fedora, but if you want the command line tools, you really need the full lm_sensors for ubuntu/debian.

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
lm-sensors seems to be [named lm_sensors](https://admin.fedoraproject.org/pkgdb/package/rpms/lm_sensors/) on fedora
also cpuburn is deprecated and is not released in ubuntu yakkety and xenial as well as in debian stretch (not sure if it exists at all in fedora), I don't know if there is a good maintained equivalent but you may consider other options if every newly released OS don't ship this package anymore.

Not: I haven't checked the arch packages availability

**Comment by tfoote** [comment]
Might stress be a possible alternative to cpuburn? It looks to be genrally available

http://packages.ubuntu.com/xenial/stress
https://packages.debian.org/stretch/stress
https://admin.fedoraproject.org/pkgdb/package/rpms/stress/

http://manpages.ubuntu.com/manpages/zesty/man1/stress.1.html

**Comment by Alessandro-Barbieri** [comment]
`app-benchmarks/cpuburn` is available in gentoo but masked in `amd64/no-multilib` profile
not available at all for non `x86` or `amd64` architectures

File: rosdep/base.yaml [hunk 1]
```diff
@@ -3144,6 +3149,11 @@ linux-headers-generic:
     wheezy: [linux-headers-3.2.0-4-all]
   fedora: [kernel-headers]
   ubuntu: [linux-headers-generic]
+lm-sensors:
+  arch: [lm_sensors]
+  debian: [lm-sensors]
+  fedora: [lm_sensors]
```
Comment (Alessandro-Barbieri) [inline rosdep/base.yaml]: add a `gentoo: [sys-apps/lm_sensors]` line

**Comment by dirk-thomas** [comment]
@mikeferguson This needs to be rebased to be mergable.

@tfoote While there might be alternative package I don't think it should prevent this patch to be merged.

**Review by tfoote — APPROVED** [review]
+1 after CI I added gentoo and resolved the conflict

**Comment by mikeferguson** [comment]
@tfoote -- thanks for helping on this -- I'm currently on the road.

**Pull request merged**